solutions/kubernetes/metrics-server/setup-metrics-server-functions.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+source /var/tmp/functions.sh
+
+function install_metricsserver() {
+    add_primary_separator "Installing Metrics Server"
+    # For Testing purpose: Fetching file from test repo, raised same file with the PR.
+    # kubectl apply -f https://raw.githubusercontent.com/seagate/cortx-re/main/solutions/kubernetes/metric-server/metric-server-components.yaml
+    kubectl apply -f https://raw.githubusercontent.com/vijwani-seagate/kubernetes/master/metric-server-components.yaml
```

Comment:
Please use CORTX_RE_REPO and CORTX_RE_BRANCH variables to generate URL dynamically